erpnext_documentation/www/docs/user/manual/en/healthcare/healthcare_settings.md

```diff
@@ -27,13 +27,19 @@ To view and change the settings, go to:
 
 * **Enable Free Follow-ups**: Many healthcare facilities do not charge for follow up consultations within a time period after the first visit (Patient Registration). Check this if you want to enable free follow-ups. After this is checked, configure the number of free follow-ups (_Patient Encounters in Valid Days_) allowed as well as the time period (_Valid number of days_) for free consultations here.
 
-## 2. Default Healthcare Service Items
+## 2. Inpatient Settings
+
+* **Allow Discharge Despite Unbilled Healthcare Services**: If this is checked then while discharging a patient, the system will not validate unbilled invoices against the patient. Some healthcare facilities discharge the patient to make the room vacant for new admissions and then generate the bill later.
+
+<img class="screenshot" alt="ERPNext Healthcare" src="{{docs_base_url}}/assets/img/healthcare/inpatient-settings.png">
```

Comment:
full image with the checkbox highlighted?

Reply:
Actually, we are writing the explanations section-wise so I have added section-wise screenshots.

Reply:
Will rectify the images in the next PR.